controllers/suite_test.go

```diff
@@ -66,7 +63,7 @@ var _ = BeforeSuite(func() {
 	Expect(err).NotTo(HaveOccurred())
 	Expect(k8sClient).NotTo(BeNil())
 
-}, 60)
+})
```

Comment:
Why did you remove`60`?

Reply:
Because it's not a valid param anymore for ginko/v2